packages/amplify-cli-core/src/feature-flags/featureFlags.ts

```diff
@@ -612,5 +612,51 @@ export class FeatureFlags {
         defaultValueForNewProjects: true,
       },
     ]);
+
+    this.registerFlag('latestRegionSupport', [
```

Comment:
Since we may run into this again as we still map several regions, I think these feature flags might make more sense as version numbers instead of booleans. That way we don't have a latestlatestPinpointRegions flag in the future.

Reply:
That's a good idea, thanks. When you say version numbers, are you just thinking an integer value that we increment each time the mappings change?

Reply:
We usually use true and false for the feature-flags correct? Or is it 1 or 0?